api/handlers.go

```diff
@@ -122,6 +128,12 @@ func (api *API) blockWithPoWPOST(w http.ResponseWriter, r *http.Request, _ httpr
 	// reporter authenticated.
 	sub := hex.EncodeToString(body.PoW.MySkyID[:])
 
+	// Check whether the skylink is on the allow list
+	if api.staticSkydAPI.IsAllowListed(r.Context(), string(body.Skylink)) {
```

Comment:
Q: I'm not sure how we add stuff to the allow list and whether there is a step where we normalise base32 skylinks to base64, for example. If there is then we're fine, if not - we should F/U.

Reply:
it's in the readme, we can F/U if necessary, we add base64 V1 skylinks